devtools/flake.nix

```diff
@@ -38,12 +38,12 @@
                     tamasfe.even-better-toml
                   ] ++ (if isAarch64Darwin then [ ] else [ vadimcn.vscode-lldb ])
                   ++ pkgs.vscode-utils.extensionsFromVscodeMarketplace [
-                     {
-                        name = "roc-lang-unofficial";
-                        publisher = "IvanDemchenko";
-                        version = "1.2.0";
-                        sha256 = "sha256-lMN6GlUM20ptg1c6fNp8jwSzlCzE1U0ugRyhRLYGPGE=";
-                      }
+                    {
+                      name = "roc-lang-unofficial";
+                      publisher = "IvanDemchenko";
+                      version = "1.2.0";
+                      sha256 = "sha256-lMN6GlUM20ptg1c6fNp8jwSzlCzE1U0ugRyhRLYGPGE=";
+                    }
```

Comment:
Were these spaces removed by a formatter?

Reply:
Yep! I ran:
```nix
nix fmt
```

And it formatted flake.nix and a file under de tools directory.